### DISCUSSION TITLE
More than one icon on holograms

### DISCUSSION THREAD

**Discussion by PoulpinSky** [Zrips/CMI#2736] [Enhancement]
If you want to set 2 icon on same line on holograms, they disapear.
Try it with ; ICON:2 ICON:1

**Cmi Version (using`/cmi version`) : 8.6.18.7** 

**Server Type (Spigot/Paperspigot/etc): Paper** 

**Server Version (using `/ver`): 1.12.2**

**Comment by Zrips** [comment]
Because its only supports one icon per line at this point. Its not a bug, its lack of feature.

**Comment by PoulpinSky** [comment]
Ok, i hope you do this soon, it will be fine for show all of my customs craft
